x-pack/plugins/ml/public/application/util/error_utils.ts

```diff
@@ -27,6 +27,13 @@ export const extractErrorMessage = (
         return error.body.message?.msg;
       }
     }
+    if (typeof error.body === 'object' && 'msg' in error.body) {
+      // @ts-ignore
+      if (typeof error.body.msg === 'string') {
+        // @ts-ignore
```

Comment:
would be great if you manage to avoid `@ts-ignore` and replace it with a type guard for instance

Reply:
I'll do this an a follow up PR